packages/manager/modules/pci/src/components/project/instance/instance.class.js

```diff
@@ -171,6 +172,10 @@ export default class Instance {
     return includes(get(this, 'image.tags', []), 'application');
   }
 
+  isDeprecatedImage() {
+    return get(this, 'image.status', '') === IMAGE_STATUS.DEPRECATED;
```

Comment:
```suggestion
    return this.image?.status === IMAGE_STATUS.DEPRECATED;
```

Reply:
Thanks @marie-j for this suggestion.
I used lodash get fn. in this case where even the `image` is not defined.
So in this case lodash check if the global path exist which i need.

I think we can use something like `this?.image?.status`, what do you think ?

Reply:
@mohammed-zahaf just `image` should be test with `?` operator. Here `this` represent the `Instance` class instance, so testing it is not required.

Reply:
Thanks for suggestions.
Updated.